calico-cloud/threat/security-anomalies.mdx

```diff
@@ -0,0 +1,82 @@
+---
+description: Machine learning to automatically alert you when clusters have security issues.
+---
+
+# Detect and alert on anomalies
+
+:::note
+
+This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
+
+:::
+
+## Big picture
+
+Generate alerts for anomalous activity within a cluster.
+
+## Value
+
+The {{prodname}} anomaly detection engine analyzes patterns and alerts on potential
+threats and issues such as:
+
+- **Security issues** - threats like DGA, reconnaissance threats like IP sweep, but also issues with servers that can affect daily operations.
+
+- **Performance issues** - spikes in data transmission, and anomalous degradation in network communication that may impact application workloads.
```

Comment:
Technically we do not have any of these types of performance anomaly detection in the new CC engine yet. We could remove this or leave it there just as a reference to different possible types of anomalies.

Reply:
Given this is tech preview, I think it is ok to leave in the statement to give some context of what the feature is trying to solve.

Reply:
This page is completely reverted now.